calico-vpp-agent/connectivity/vxlan.go

```diff
@@ -141,8 +139,7 @@ func (p *VXLanProvider) AddConnectivity(cn *common.NodeConnectivity) error {
 		return err
 	}
 	if cn.Vni != 0 {
-		_, found := p.netsLoopbacks[cn.Vni]
-		if !found {
+		if !p.server.networks[cn.Vni].LoopbackConfigured {
```

Comment:
Couldn't we move all this code to `calico-vpp-agent/cni/pod_interface/loopback.go:CreateInterface()` ? That would remove the need to add `LoopbackConfigured` on the network object and updating state here

Reply:
that one is specific for pod interface loopback, here we have a loopback for the secondary network tunnel. Plus `/cni/pod_interface` is specific for cni and pods. Do you mean we create a common function that creates loopbacks for pods interfaces and for networks vxlans and put somewhere in a common place?

Reply:
Oh sorry, I meant in `CreateVRFsForNet()`, I got confused by searching for `LoopbackSwIfIndex =`